pyQuARC/code/checker.py

```diff
@@ -52,8 +54,8 @@ def __init__(
             self.checks,
             self.checks_override
         )
-        self.schema_validator = SchemaValidator(self.messages, metadata_format)
-        self.tracker = Tracker(self.rule_mapping, self.rules_override)
+        self.schema_validator = SchemaValidator(metadata_format)
+        self.tracker = Tracker(self.rule_mapping, self.rules_override, metadata_format)
```

Comment:
Maybe change format to keyword arg?

Reply:
https://github.com/NASA-IMPACT/pyQuARC/pull/84/commits/878b39842d29566c68615507c820f90af9ee6acf